ethers-middleware/tests/it/wallets.rs

```diff
@@ -0,0 +1,62 @@
+static WALLETS: Lazy<TestWallets> = Lazy::new(|| {
```

Comment:
wait how is this working if it's importing nothing?

Reply:
oh yeah i didn't want to delete it, but it's not used anywhere anymore since all middleware tests are on anvil now

Reply:
ack - let's get rid of then